### ISSUE TITLE
[Bug]: Storybook asks to send Crash report on all errors

### ISSUE THREAD

**Issue by fireflysemantics** [storybookjs/storybook#22510]
### Describe the bug

When there's an error and we hit ctrl c to fix it and restart, storybook still asks if we want to send a crash report so we have to hit ctrl c again ... It would be nice if it only asked about crash reports when it crashes outside of normal errors:

```
✖ Would you like to help improve Storybook by sending anonymous crash reports? … yes
^C
```

### System

```shell
Angular 15
```

**Comment by shilman** [comment]
If you respond "n" it shouldn't ask you again until you clear your `node_modules`. Does it?

**Comment by nextlevelshit** [comment]
Where is the configuration to turn this off by default?

**Comment by dddcxy** [comment]
I would also like to know where is the configuration to turn this off by default?

**Comment by dddcxy** [comment]
> 我还想知道默认关闭此功能的配置在哪里？

i found it [https://storybook.js.org/docs/react/configure/telemetry#crash-reports-disabled-by-default](url)

**Comment by scmx** [comment]
new link https://storybook.js.org/docs/react/configure/telemetry#crash-reports-disabled-by-default
```javascript
core: {
        enableCrashReports: false,
},
```

**Comment by httran13** [comment]
To disable this prompt entirely add this flag --disable-telemetry

This will prevent the prompt that causes your ci server to hang

**Comment by maxpatiiuk** [comment]
We are using turbo build, so the CI doesn't print logs until command failed or succeeded.
Storybook's "Would you like to help improve Storybook by sending anonymous crash reports" hanged the CI machine, and we did not see anything in the logs since our turbo config doesn't print until process exits.

Took a while to debug that Storybook is the culprit

Doing interactive prompts in non-interactive environment (CI machine) is a bug - Storybook should not do that

**Comment by maxpatiiuk** [comment]
> If you respond "n" it shouldn't ask you again until you clear your node_modules. Does it?

@shilman CI machines clear node_modules on each run

**Comment by shilman** [comment]
@tmeasday any recollection why we added this extra clause?

https://github.com/storybookjs/storybook/commit/090fb358acabcab5a73aeeea2e85a7130a8bbc4f?diff=unified&w=0

**Comment by tmeasday** [comment]
I don't remember but I assume because we were trying to test the prompting. Perhaps we should use our own env var for this or something,.

**Comment by yannbf** [comment]
This should be fixed in our upcoming Storybook 8.2 release!
